### PR TITLE
Node documentation improvements

### LINKED ISSUE #864: Improve node documentation
The run a windows node needs to make more explicit about the backup of the toml file before the upgrade. Also, it needs to be clearer in the windows node downloads about the toml file. And the sections about node configuration need to be consistent...it needs to be earlier and also clearer in e.g. enable inbound connections.

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Clarify some of the steps and simplify the Windows node documentation.

## Changes

Update the windows node documentation and node runner service documentation.
Add slashes to iOS code examples where path is two words.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.
Closes #864

### CODE REVIEW

File: source/mainnet/net/nodes/baker-macos.rst [hunk 1]
```diff
@@ -34,11 +34,11 @@ On mainnet
 
    .. code-block:: console
 
-      $sudo cp "/path/to/mainnet/baker-credentials.json" "/Library/Application Support/Concordium Node/Mainnet/Config/baker-credentials.json"
+      $sudo cp "/path/to/mainnet/baker-credentials.json" "/Library/Application\ Support/Concordium\ Node/Mainnet/Config/baker-credentials.json"
```
Comment (nicoleta-andreiana) [inline source/mainnet/net/nodes/baker-macos.rst:37]: the backslash from "Application Support" needs to be removed. It should only be part of the "Concordium Node" folder.

File: source/mainnet/net/nodes/baker-macos.rst [hunk 2]
```diff
@@ -57,11 +57,11 @@ On testnet
 
    .. code-block:: console
 
-      $sudo cp "/path/to/testnet/baker-credentials.json" "/Library/Application Support/Concordium Node/Testnet/Config/baker-credentials.json"
+      $sudo cp "/path/to/testnet/baker-credentials.json" "/Library/Application\ Support/Concordium\ Node/Testnet/Config/baker-credentials.json"
```
Comment (nicoleta-andreiana) [inline source/mainnet/net/nodes/baker-macos.rst:60]: The backslash from. "Application Support" needs to be removed.

File: source/mainnet/net/nodes/node-runner-service-configuration.rst [hunk 3]
```diff
@@ -5,7 +5,9 @@
 Concordium Windows node runner service configuration
 ====================================================
 
-The Concordium Node Runner Service ("service" for short) is configured using a TOML file. This file is typically located at ``C:\ProgramData\Concordium\Node Runner\nodes.toml.`` (The path on your system is determined by the ``Config`` value in the registry key ``HKEY_LOCAL_MACHINE\SOFTWARE\Concordium\Node Runner``.)
+You can change the mainnet or testnet node configuration with the Configure Concordium Node Service. The Concordium Node Runner Service ("service" for short) is configured using a TOML file. This file is typically located at ``C:\ProgramData\Concordium\Node Runner\nodes.toml.`` (The path on your system is determined by the ``Config`` value in the registry key ``HKEY_LOCAL_MACHINE\SOFTWARE\Concordium\Node Runner``.)
+
+To run the app, search for *configure concordium node* in the **Search** bar, and then select **Configure Concordium Node Service**. If you see a message saying *Do you want to allow this app to make changes to your device?*, select **Yes**. The configuration file opens in Notepad.
```
Comment (abizjak) [inline source/mainnet/net/nodes/node-runner-service-configuration.rst:10]: I think this last part "opens in Notepad" is technically incorrect. If they have set some other editor to open `toml` files then it'll be that app that opens.

But it's probably better to leave it like this. If people have editors to edit TOML files then they'll know what's going on I imagine.

**Review by abizjak — APPROVED** [review]

File: source/mainnet/net/nodes/baker-macos.rst (hunk 1 above)
Comment (mh-concordium) [reply source/mainnet/net/nodes/baker-macos.rst]: Why is that?
I thought that if there should be a space in the path then backslash should stay as it is an escape character. Similar to Concordium\ Node.

**Review by mh-concordium — COMMENTED** [review]
Looks good.

File: source/mainnet/net/nodes/baker-macos.rst (hunk 1 above)
Comment (nicoleta-andreiana) [reply source/mainnet/net/nodes/baker-macos.rst]: From what I noticed is that "Application Support" being an OS defined folder, it's accepted without a backslash . If we use the backslash there, the folder will not be found and another one will be created. This doesn't apply for "Concordium\Node" as it's a folder we are creating.

File: source/mainnet/net/guides/run-node-macos.rst [hunk 4]
```diff
@@ -209,27 +209,27 @@ If you want to change whether the node services start automatically, you have tw
 
       .. code-block:: console
 
-          $sudo ln -s "/Library/Concordium Node/LaunchDaemons/software.concordium.mainnet.node.plist" "/Library/LaunchDaemons/"
+          $sudo ln -s "/Library/Concordium\ Node/LaunchDaemons/software.concordium.mainnet.node.plist" "/Library/LaunchDaemons/"
```
Comment (mh-concordium) [inline source/mainnet/net/guides/run-node-macos.rst:212]: When using quotations, it is not needed to escape.
```suggestion
          $sudo ln -s "/Library/Concordium Node/LaunchDaemons/software.concordium.mainnet.node.plist" "/Library/LaunchDaemons/"
```

File: source/mainnet/net/guides/run-node-macos.rst [hunk 5]
```diff
@@ -209,27 +209,27 @@ If you want to change whether the node services start automatically, you have tw
...
 
     - For testnet:
 
       .. code-block:: console
 
-          $sudo ln -s "/Library/Concordium Node/LaunchDaemons/software.concordium.testnet.node.plist" "/Library/LaunchDaemons/"
+          $sudo ln -s "/Library/Concordium\ Node/LaunchDaemons/software.concordium.testnet.node.plist" "/Library/LaunchDaemons/"
```
Comment (mh-concordium) [inline source/mainnet/net/guides/run-node-macos.rst:218]: When using quotations, it is not needed to escape.
```suggestion
          $sudo ln -s "/Library/Concordium Node/LaunchDaemons/software.concordium.testnet.node.plist" "/Library/LaunchDaemons/"
```

File: source/mainnet/net/guides/run-node-macos.rst [hunk 6]
```diff
@@ -209,27 +209,27 @@ If you want to change whether the node services start automatically, you have tw
...
 
   - Enable automatic startup of the *node-collector* by running:
 
     - For mainnet:
 
       .. code-block:: console
 
-         $sudo ln -s "/Library/Concordium Node/LaunchDaemons/software.concordium.mainnet.node-collector.plist" "/Library/LaunchDaemons/"
+         $sudo ln -s "/Library/Concordium\ Node/LaunchDaemons/software.concordium.mainnet.node-collector.plist" "/Library/LaunchDaemons/"
```
Comment (mh-concordium) [inline source/mainnet/net/guides/run-node-macos.rst:226]: When using quotations, it is not needed to escape.
```suggestion
         $sudo ln -s "/Library/Concordium Node/LaunchDaemons/software.concordium.mainnet.node-collector.plist" "/Library/LaunchDaemons/"
```

File: source/mainnet/net/guides/run-node-macos.rst [hunk 7]
```diff
@@ -209,27 +209,27 @@ If you want to change whether the node services start automatically, you have tw
...
 
     - For testnet:
 
       .. code-block:: console
 
-         $sudo ln -s "/Library/Concordium Node/LaunchDaemons/software.concordium.testnet.node-collector.plist" "/Library/LaunchDaemons/"
+         $sudo ln -s "/Library/Concordium\ Node/LaunchDaemons/software.concordium.testnet.node-collector.plist" "/Library/LaunchDaemons/"
```
Comment (mh-concordium) [inline source/mainnet/net/guides/run-node-macos.rst:232]: When using quotations, it is not needed to escape.
```suggestion
         $sudo ln -s "/Library/Concordium Node/LaunchDaemons/software.concordium.testnet.node-collector.plist" "/Library/LaunchDaemons/"
```

**Review by mh-concordium — CHANGES_REQUESTED** [review]
I looked it again.

**Review by mh-concordium — APPROVED** [review]
looks good to me.

**Pull request merged**